examples/bunny-twirl/index.js

```diff
@@ -56,13 +56,26 @@ container.addChild(text)
 let isAnimating = true
 let isPlaying = false
 
+// Generate map
+const challengeNumber = Rune.getChallengeNumber();
+// const nextChallengeNumber = challengeNumber + 1;
+
+// 1. Game map randomization
+const random = new Math.seedrandom(challengeNumber)
+const pseudoRandomNumber = random()
+const speed = (pseudoRandomNumber / 100) * 3 + 0.005
+
+// 2. Fixed list of game maps
```

Comment:
Similar to my other comment, should we add something like `// Derive direction (fixed list) from Rune challenge number`?

Reply:
Fixed in https://github.com/rune/rune-games-sdk/pull/24/commits/75d27bf8f857bf99a576dc5fd7abcb06909a0d51